sync/awsdatabricks.py

```diff
@@ -252,20 +251,21 @@ def get_prediction_job(
         job = get_default_client().get_job(job_id)
         if "error_code" in job:
             return Response(error=DatabricksAPIError(**job))
-
```

Comment:
Is this intentional? I thought the line break helped set apart the error handling/early exit from the meat of the function.

Reply:
Think this was accidental when I was mucking about. Reverted.